### DISCUSSION TITLE
Process community rewards cycle: 9/23/23 - 10/20/23

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#819] [Rewards]
Another cycle has passed, so it is once again time to distribute rewards to our community contributors based on our newly refined measurements listed in the bottom of the page of our [Ongoing Contributions spreadsheet](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=238932857):

```
# NOTE ASA Stats Community rewards 9/23/23 - 10/20/23
AlgoRhythMatic 0.07 damo
Algoworm 0.48 damo
DingusMungus 0.09 damo
django.algo 0.03 damo
dragmz  0.14 damo
garywontwin 0.06 damo
kerrilija 0.96 damo
Maximus 0.12 damo
Milesmile 0.12 damo
Mocha  0.06 damo
patrick.algo 0.06 damo
robbie 0.06 damo
u/centrips 0.03 damo
UncleDooom 0.03 damo
TOTAL 2.31 damo
```
Please review and comment if there are any questions or concerns. The rewards noted above for this period use newly-refined rewards values discussed here: https://github.com/asastats/channel/discussions/405

Per usual process, we'll wait ~24 hrs for comments/feedback, and then pass on the list to the rewards sender (currently @ipaleka).

As well, please comment or reach out to me if you are interested in participating in rewards measurements in future cycles!

**Comment by ipaleka** [comment]
Great job as always, thanks! :)
Please if you can add this too as I forgot what date is and didn't close it on time: https://github.com/asastats/channel/issues/763

**Reply by AlgoRhythMatic** [reply]
Done!  Updated article above to adjust count.
